base/dates/parse.jl

```diff
@@ -125,9 +125,9 @@ Returns a 2-element tuple `(values, pos)`:
   the passed in type.
 * `pos::Int`: The character index at which parsing stopped.
 """
-@generated function tryparsenext_internal{T<:TimeType}(
+@generated function tryparsenext_internal(
     ::Type{T}, str::AbstractString, pos::Int, len::Int, df::DateFormat, raise::Bool=false,
-)
+) where T<:TimeType
```

Comment:
@tkelman  the line breaking here is pretty unconventional from the rest of base.  Any suggestions.